app/pages/_document.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import Document, {
+  Html,
+  Head,
+  Main,
+  NextScript,
+} from 'next/document';
+
+function Analytics() {
+  return (
+    <>
+      <script async src={`https://www.googletagmanager.com/gtag/js?id=${process.env.GOOGLE_ANALYTICS}`} />
+      <script
+        dangerouslySetInnerHTML={{
+          __html: `
+            window.dataLayer = window.dataLayer || [];
+            function gtag() {
+              dataLayer.push(arguments);
+            }
+            gtag('js', new Date());
+            gtag('config', ${process.env.GOOGLE_ANALYTICS});
+          `,
+        }}
+      />
+    </>
+  );
+}
+
+function Rollbar() {
+  return (
+    <>
+      <script
+        dangerouslySetInnerHTML={{
+          __html: `
+            var _rollbarConfig = {
+              accessToken: ${process.env.ROLLBAR_API_KEY},
```

Comment:
I've specified ROLLBAR_API_KEY=awikey and got:
![image](https://user-images.githubusercontent.com/14346393/115375789-cc719080-a1d6-11eb-8c47-ba4719a81175.png)

It seems "" are missing.